examples/plot_extra_oblique_random_forest.py

```diff
@@ -0,0 +1,142 @@
+"""
+===============================================================================
+Plot extra oblique forest and oblique random forest predictions on cc18 datasets
+===============================================================================
+
+A performance comparison between extra oblique forest and standard oblique random
+forest using three datasets from OpenML benchmarking suites.
+
+Extra oblique forest uses Extra-trees as base model which differ from classic
```

Comment:
Extra oblique forests uses extra oblique trees as base model, not extra-trees. Extra-trees in scikit-learn corresponds to axis-aligned splits, whereas oblique trees correspond to oblique splits (sparse linear combinations of features)